### ISSUE TITLE
Ignore pods by label/name pattern

### ISSUE THREAD

**Issue by codekoala** [abahmed/kwatch#240]
**Is your feature request related to a problem? Please describe.**
I would like to ignore alerts from pods that are part of helm charts that I do not have control over. Sometimes these pods include containers with very generic names such as `main` or `app`, which make it difficult to use `ignoreContainerNames` for targeting these specific pods while continuing to receive alerts for other pods for our own apps with containers having those same names. 

**Describe the solution you'd like**
I think something similar to this would be useful:

```yaml
ignorePodNames:
- some-pod-*
```
or maybe regex:
```yaml
ignorePodNamePatterns:
- some-pod-.*
```
Perhaps something involving pod labels:
```yaml
ignorePodLabels:
- label: something
  value: a value
- label: another-selector
  value: pattern-*
```

I feel like the name/name pattern based option would be much more straightforward, while the label-based option would be very helpful for a lot of different scenarios. I could see the label-based option getting complicated though (must a pod match _all_ labels before being ignored? must one pod match multiple labels but we still need to add other labels to ignore separate pods?).

**Comment by alikoptan** [comment]
Hey @abahmed, I would like to try and work on this :)

**Comment by fschlager** [comment]
Sorry for cutting in line, but I really needed this in production.

**Comment by abahmed** [comment]
Released in [v0.8.5](https://github.com/abahmed/kwatch/pkgs/container/kwatch) 🎉
